platforms-serverless/netlify-github/package.json

```diff
@@ -9,8 +9,8 @@
     "ts-node": "10.9.1"
   },
   "scripts": {
-    "dev": "npx netlify dev",
-    "build": "PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platform netlify-github build' && yarn prisma generate"
+    "dev": "pnpm netlify dev",
+    "build": "PRISMA_TELEMETRY_INFORMATION='ecosystem-tests platform netlify-github build' && pnpm prisma generate"
```

Comment:
Third alterantive to `postinstall` or setting build script via UI I guess?

Reply:
(can this be marked as resolved?)